assets/app/view/game/buy_company_from_other_player.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
```

Comment:
need to have require actionable to be safe here

Reply:
Fixed